test/integ/test_connection.py

```diff
@@ -1060,3 +1060,13 @@ def test_client_prefetch_threads_setting(conn_cnx):
             cur.execute(f"alter session set client_prefetch_threads={new_thread_count}")
             assert cur._result_set.prefetch_thread_num == new_thread_count
         assert conn.client_prefetch_threads == new_thread_count
+
+
+@pytest.mark.aws
+@pytest.mark.external
+def test_client_failover_connection_url(conn_cnx):
+    with conn_cnx("client_failover") as conn:
```

Comment:
This connection is configured using the client_failover parameters? Are those values in this PR or are they defined elsewhere/outside of github?

Reply:
They are in this PR, they are just encrypted